Service/RefreshToken.php

```diff
@@ -85,18 +84,16 @@ public function __construct(
      */
     public function refresh(Request $request)
     {
+        $credentials = $this->authenticator->getCredentials($request);
+
         try {
-            $user = $this->authenticator->getUser(
-                $this->authenticator->getCredentials($request),
-                $this->provider
-            );
+            $user = $this->authenticator->getUser($credentials, $this->provider);
 
             $postAuthenticationToken = $this->authenticator->createAuthenticatedToken($user, $this->providerKey);
```

Comment:
this should probably removed or deprecated too, as `PostAuthenticationGuardToken` is deprecated.
For `->onAuthenticationSuccess()` any `TokenInterface` will work, as it only uses `getUser()`.
And for `RefreshToken` event getting pre authenticated toeken should probably be deprecated or reworked.. I'm not sure why is it needed at all